runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -20,12 +20,13 @@ pressing [<Esc>](<Esc>) and then [u](u) will undo the latest change.
 This tutorial is interactive, and there are a few things you should know.
 - Type [<Enter>](<Enter>) on links [like this](holy-grail    ) to open the linked help section.
 - Or simply type [K](K) on any word to find its documentation!
+- You can close this help split with `:q`{vim}
```

Comment:
maybe "help window" is better? "split" is a technical Vim term that beginners are likely not familiar with